protocol/messenger_status_updates.go

```diff
@@ -267,3 +267,63 @@ func (m *Messenger) HandleStatusUpdate(state *ReceivedMessageState, statusMessag
 func (m *Messenger) StatusUpdates() ([]UserStatus, error) {
 	return m.persistence.StatusUpdates()
 }
+
+func (m *Messenger) timeoutStatusUpdates(fromClock uint64, tillClock uint64) {
+	// Most of the time we only need to time out just one status update,
+	// but the range covers special cases like, other status updates had the same clock value
+	// or the received another status update with higher clock value than the reference clock but
+	// lower clock value than the statusUpdate.Clock
+	deactivatedStatusUpdates, err := m.persistence.DeactivatedStatusUpdates(fromClock, tillClock)
```

Comment:
I think that if `persistence` returns an error it would be useful to know what that error is. Otherwise this function will just silently fail